app/api/server/v1/rooms.js

```diff
@@ -535,7 +535,9 @@ API.v1.addRoute(
 				dateFrom = new Date(dateFrom);
 				dateTo = new Date(dateTo);
 				dateTo.setDate(dateTo.getDate() + 1);
-
+				if (dateFrom > dateTo) {
+					throw new Meteor.Error('error-invalid-dates');
```

Comment:
Hey! I would recommend you to import TAPi18n from 'meteor/rocketchat:tap-i18n' and call for the translation of error-invalid-dates. That way, users won't see 'error-invalid-dates' spawning in their screens but actually a more helpful message in their native language explaining what caused the error.